bootstrap/src/test/java/io/airlift/bootstrap/TestBootstrap.java

```diff
@@ -111,8 +110,8 @@ public void testStrictConfig()
                 .setRequiredConfigurationProperty("test-required", "foo");
 
         assertThatThrownBy(bootstrap::initialize)
-                .isInstanceOfSatisfying(CreationException.class, e ->
-                        assertThat(e.getErrorMessages()).containsExactly(
+                .isInstanceOfSatisfying(io.airlift.bootstrap.ConfigurationException.class, e ->
```

Comment:
This shows the name collision I mentioned above. But this is test code, maybe doesn't matter in real usage.